static/app/components/events/breadcrumbs/breadcrumbsDrawer.tsx

```diff
@@ -265,55 +273,6 @@ const VisibleFocusButton = styled(Button)`
     0 0 1px;
 `;
 
-const BreadcrumbDrawerContainer = styled('div')`
```

Comment:
moved to a new file, `eventDrawer.tsx` which has all the shared components for the breadcrumbs + flag drawers